Standardize external replication (ETL) docs (#46875)
supabase/supabase: apps/docs/content/guides/database/postgres/roles.mdx

### `supabase_etl_admin`

> Used by [external replication (ETL)](/docs/guides/database/replication), powered by Supabase ETL, to replicate database changes to destination systems. Has read-all access and replication privileges for change data capture, bypasses Row Level Security, and can write to the `etl` schema.

Reviewer: I'm a little confused here. Are we trying to say…

```suggestion
Used by [external replication (ETL)](/docs/guides/database/replication), which is powered by Supabase ETL, to replicate database changes to destination systems. Has read-all access and replication privileges for change data capture, bypasses Row Level Security, and can write to the `etl` schema.
```

Maybe?

Reviewer 2: I agree with @iambriccardo that the "by Supabase" should be taken outside of the link. Links should be explicit about where they are going, which is why the title of the document is the best choice.

**Source:** [Link Purpose (In Context) (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context)

I have a suggestion below for a revision, but I realize it can make it inconsistent with the others. A quick polish to make each role consistent would be nice. 😄  (Feel free to break out a Linear task in Docs if out of scope)

```suggestion
`supabase_etl_admin` is used by [Database replication](/docs/guides/database/replication) through Supabase ETL. 

This role:

- Replicates database changes to destination systems

- Has read-all access

- Has replication privileges for change data capture, bypasses Row Level Security

- Can write to the `etl` schema
```

Changes and reasons:
- **Change link to title of document:** A standard convention so that readers can consistently know what to expect when clicking a link.
- **Repeat the role:** Avoid multiple sentence fragments and increase clarity by repeating the name of the role.
- **Remove 'powered by':** From a localization perspective, this is indirect, metaphorical language. "through" is more direct.
- **Bullet list added:** Easier readability
